packages/lit-element/src/private-ssr-support.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {$private as p} from './lit-element.js';
```

Comment:
I realize you tried to explain this in the PR comments, but I think this module needs some comments or we'll forget why it's being done this circuitous way.

Reply:
Whoops, this is intended to just copy the pattern already established in `lit-html`; I added the comment from the same file in `lit-html`:
```js
/**
 * END USERS SHOULD NOT RELY ON THIS OBJECT.
 *
 * We currently do not make a mangled rollup build of the lit-ssr code. In order
 * to keep a number of (otherwise private) top-level exports  mangled in the
 * client side code, we export a $private object containing those members (or
 * helper methods for accessing private fields of those members), and then
 * re-export them for use in lit-ssr. This keeps lit-ssr agnostic to whether the
 * client-side code is being used in `dev` mode or `prod` mode.
 */
```

Reply:
Let's try to change this to `_private` instead of `$private` to match our convention for private (assuming top level exports don't get mangled).